src/commands/cloud.rs

```diff
@@ -0,0 +1,26 @@
+use clap::Subcommand;
+use anyhow::Result;
+
+use super::deploy::DeployCommand;
+use super::login::LoginCommand;
+
+/// Commands for publishing applications to Fermyon Cloud.
+#[derive(Subcommand, Debug)]
+pub enum CloudCommands {
+    /// Package and upload to Fermyon Cloud
+    #[clap(alias = "deploy")]
```

Comment:
The alias doesn't seem to work for me, as it seems to expect a plugin:

```
$ spin deploy       
Unknown command: plugin 'deploy' not found at expected location /Users/radu/Library/Application Support/spin/plugins/manifests/deploy.json: No such file or directory (os error 2)

$ spin login       
Unknown command: plugin 'login' not found at expected location /Users/radu/Library/Application Support/spin/plugins/manifests/login.json: No such file or directory (os error 2
```

Reply:
Yeah this was part of an earlier experimentation that does not work.

See https://github.com/fermyon/spin/issues/1117#issuecomment-1428908182 for more details.